clients/src/main/java/org/apache/kafka/clients/admin/DeleteConsumerGroupsResult.java

```diff
@@ -29,13 +29,13 @@
  */
 @InterfaceStability.Evolving
 public class DeleteConsumerGroupsResult {
-    final KafkaFuture<Map<String, KafkaFuture<Void>>> futures;
+    final Map<String, KafkaFuture<Void>> futures;
 
-    DeleteConsumerGroupsResult(KafkaFuture<Map<String, KafkaFuture<Void>>> futures) {
+    DeleteConsumerGroupsResult(final Map<String, KafkaFuture<Void>> futures) {
         this.futures = futures;
     }
 
-    public KafkaFuture<Map<String, KafkaFuture<Void>>> deletedGroups() {
+    public Map<String, KafkaFuture<Void>> deletedGroups() {
         return futures;
     }
 }
```

Comment:
Hmm, probably there should be an `all` method, for consistency with the other admin client result types